wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,442 @@
+# FoundryVTT dnd5e Active Effects Examples
+
+![Up to date as of 2.3.x](https://img.shields.io/badge/dnd5e-v2.3.x-informational)
+
+This document only covers Active Effects available to the Core dnd5e System.
+
+## Legend
+
+`[number]` - These square brackets mean "replace this with your value of the type within the brackets".
+So this example: `+[number]` would mean you input `+3`.
+
+`[formula]` - When `formula` is mentioned in this document it means this value can be populated with any dice formula. For example, Bless adds several effects with the Effect Value of `1d4`.
+
+As part of this, an [Actor's Rolldata](https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas) is available as "@attributes." Useful examples:
+
+| @attribute                 | Description                   |
+| -------------------------- | ----------------------------- |
+| `@abilities.dex.mod`       | Actor's Dexterity Modifier    |
+| `@prof`                    | Actor's Proficiency Bonus     |
+| `@details.level`           | Actor's overall Level         |
+| `@classes.barbarian.levels` | Actor's Barbarian Class Level |
+
+> [!Note]
+> When using formulas in an Active Effect Value, the actor sheet display that corresponds to the changed value will not display the evaluated formula, but it will be applied when rolled.
+> E.g. When adding `+@abilities.cha.mod` to `system.bonuses.abilities.save` to simulate a Paladin's Aura of Protection, the actor sheet will not display that bonus applied to saving throws. The bonus will be present when the saving throw is rolled.
+
+| Change Mode | Description |
+|------------ | ------------|
+| Add         | Adds the provided value to a number. This can be used to both add and subtract from a particular value by specifying `+1` or `-1` as the value to add. |
```

Comment:
This works on 'array' or 'set' values. We even have some examples in this wiki like adding a condition type. I'm not sure how best to explain that though as we shouldn't presume the average user understands what an 'array' or 'set' is.